app/Resources/SonataAdminBundle/views/CRUD/base_array_macro.html.twig

```diff
@@ -1,12 +1,5 @@
 {#
-
-This file is part of the Sonata package.
-
-(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
-
-For the full copyright and license information, please view the LICENSE
-file that was distributed with this source code.
-
+    Override sonata template to have something more readable.
```

Comment:
Nice, I would have love more informations : the old one is just a coma separated list while this one is providing HTML rendering blablablablablablabla.